tests/wpt/wpt.ts

```diff
@@ -689,7 +689,8 @@ function createReportTestCase(expectation: boolean | string[]) {
     switch (status) {
       case 0:
         if (expectFail) {
-          simpleMessage += red("ok (expected fail)");
```

Comment:
This being all in red made it sound like it was bad that it passed.

Reply:
Maybe change to `ok (previously fail)`? The simple message is no longer distinguishable from tests that were supposed to pass.

Reply:
I accidentally did a semi-colon after the " ".

Reply:
Should be better now.